src/User32.Desktop/User32.cs

```diff
@@ -144,6 +159,12 @@ public static partial class User32
         /// </returns>
         public delegate IntPtr DialogProc(IntPtr hwndDlg, WindowMessage uMsg, IntPtr wParam, IntPtr lParam);
 
+        public delegate void MsgBoxCallback(HELPINFO lpHelpInfo);
+
+        public delegate bool MonitorEnumDelegate(IntPtr hMonitor, IntPtr hdcMonitor, ref RECT lprcMonitor, IntPtr dwData);
```

Comment:
Please rename to `MonitorEnumProc` and change last two parameter types to `RECT*` and `void*` respectively.